setup.py

```diff
@@ -43,20 +43,29 @@ def maybe_require(pkg):
 ])
 
 develop = set([
-    'flake8==3.3.0',
-    'coverage==4.3.4',
-    'pytest==3.0.6',
-    'pytest-cov==2.4.0',
-    'mock==2.0.0',
-    'Sphinx',
-    'nbsphinx==0.3.1',
-    'sphinx_rtd_theme',
     'futures==3.0.5',
     'requests==2.13.0',
     'wheel',
+])
+
+docs = set([
+    'Sphinx',
+    'nbsphinx==0.3.1',
+    'sphinx_rtd_theme',
     'ipython<6',
 ])
 
+testing = set([
+    'coverage==4.3.4',
+    'pytest==3.0.6',
+    'pytest-cov==2.4.0',
+    'mock==2.0.0',
+])
+
+flake8 = set([
```

Comment:
I’d give this group some more generic name, something like _linting_. The other groups are not named by the specific packages too.

Moreover as [you informed](
https://github.com/RedHatInsights/insights-core/pull/1335#issuecomment-412646635), there might be issues with running Flake8 with Python2.6. @eduardocerqueira suggested switching to a different linter that’d be compatible. If this happens, the name would be obsolete, but a more generic name could stay the same.

Reply:
@Glutexo that makes sense to me too, I will switch to `linting`.